tests/test_translate.py

```diff
@@ -204,9 +206,9 @@ def test_sequential_savepoint(
                 print_failures,
                 failure_stride,
                 test_name,
-                ignore_near_zero,
-                testobj.near_zero,
-                xy_indices,
+                ignore_near_zero_errors=ignore_near_zero,
```

Comment:
This is updated because I was getting weird behaviors like variable values not propagating through this call. I wanted to be sure they weren't getting crossed.